Demo.Api/Migrations/20240418184813_initial.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+using Nethereum.Util;
+using Npgsql.EntityFrameworkCore.PostgreSQL.Metadata;
+
+#nullable disable
+
+namespace Demo.Api.Migrations
```

Comment:
We shouldn't include the migrations to the demo to
* Allow users to run their own migrations
* Keep the demo lightweight
What do you think?